az-digital/az_quickstart
# Closes #2671: Enable copy options based on content type

## Description
Make "Copy email snippet" options pertinent to content type. Removed fields that were left in case they were needed for this purpose.

## Related issues
Closes #2671

## How to test
1. Enable Quickstart for Marketing Cloud
2. Verify email and news copy links show as expected
3. Ensure copy functionality still works

## Types of changes


### Arizona Quickstart (install profile, custom modules, custom theme)
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal core
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

### Drupal contrib projects
- **Patch release changes**
- **Minor release changes**
- **Other or unknown**

## Checklist
